engine.py

```diff
@@ -197,6 +217,84 @@ def log_error(self, msg):
         
     
     ##########################################################################################
+    # panel interfaces
+
+    def show_panel(self, panel_id, title, bundle, widget_class, *args, **kwargs):
+        """
+        Shows a panel in a way suitable for this engine. The engine will attempt to
+        integrate it as seamlessly as possible into the host application. If the engine does 
```

Comment:
That second sentence is superfluous, we know that the Nuke engine does support panels.